pkg/cmd/roachtest/BUILD.bazel

```diff
@@ -53,14 +53,12 @@ go_binary(
 
 go_test(
     name = "roachtest_test",
-    size = "small",
     srcs = [
         "z_cluster_test.go",
         "z_test_registry_test.go",
         "z_test_test.go",
     ],
     embed = [":roachtest_lib"],
-    tags = ["broken_in_bazel"],
```

Comment:
Careful -- the removal of this tag started breaking Bazel CI runs. :)

Reply:
Interesting, thanks for fixing. Odd enough I think the tag got removed by `make bazel-generate`. I'll keep this in mind for the future.